includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -753,44 +754,46 @@ public function wc_setup_shipping() {
 		<h1><?php esc_html_e( 'Shipping', 'woocommerce' ); ?></h1>
 		<p><?php echo esc_html( $intro_text ); ?></p>
 		<form method="post">
-			<ul class="wc-wizard-services shipping">
-				<li class="wc-wizard-service-item">
-					<div class="wc-wizard-service-name">
-						<p><?php echo esc_html_e( 'Shipping Zone', 'woocommerce' ); ?></p>
-					</div>
-					<div class="wc-wizard-service-description">
-						<p><?php echo esc_html_e( 'Shipping Method', 'woocommerce' ); ?></p>
-					</div>
-				</li>
-				<li class="wc-wizard-service-item">
-					<div class="wc-wizard-service-name">
-						<p><?php echo esc_html( $country_name ); ?></p>
-					</div>
-					<div class="wc-wizard-service-description">
-						<?php $this->shipping_method_selection_form( $country_code, 'shipping_zones[domestic]' ); ?>
-					</div>
-					<div class="wc-wizard-service-enable">
-						<span class="wc-wizard-service-toggle">
-							<input id="shipping_zones[domestic][enabled]" type="checkbox" name="shipping_zones[domestic][enabled]" value="yes" checked="checked" />
-							<label for="shipping_zones[domestic][enabled]">
-						</span>
-					</div>
-				</li>
-				<li class="wc-wizard-service-item">
-					<div class="wc-wizard-service-name">
-						<p><?php echo esc_html_e( 'Locations not covered by your other zones', 'woocommerce' ); ?></p>
-					</div>
-					<div class="wc-wizard-service-description">
-						<?php $this->shipping_method_selection_form( $country_code, 'shipping_zones[intl]' ); ?>
-					</div>
-					<div class="wc-wizard-service-enable">
-						<span class="wc-wizard-service-toggle">
-							<input id="shipping_zones[intl][enabled]" type="checkbox" name="shipping_zones[intl][enabled]" value="yes" checked="checked" />
-							<label for="shipping_zones[intl][enabled]">
-						</span>
-					</div>
-				</li>
-			</ul>
+			<?php if ( empty( $existing_zones ) ) { ?>
```

Comment:
Use PHP alternate syntax when mixing with HTML.

`<?php if ( empty( $existing_zones ) ) : ?>`
...
`<?php endif; ?>`

Reply:
Changed in https://github.com/woocommerce/woocommerce/pull/16995/commits/f2d05be59b23b21c5d5f5281b9ae891b146e7e58